py/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -240,6 +246,8 @@ def __init__(
             else:
                 self.close()
                 raise Exception("unexpected response")
+        if remote_static_key not in remote_keys:
+            remote_keys.append(remote_static_key)
```

Comment:
this does not persist it anywhere. maybe it would be easier to copy how it is done in bitbox02-api-go, e.g. pass an instance with some functions for getting/setting keys?

Reply:
I think the application (not the communication library) should store any state. So I think this PR is correct.